lambda_function/aws/resources/load_balancer_handler.py

```diff
@@ -0,0 +1,96 @@
+import json
+import logging
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import boto3
+import consts
+from aws.resources.base_handler import BaseHandler
+from port.entities import create_entities_json, handle_entities
+
+logger = logging.getLogger(__name__)
+
+
+class LoadBalancerHandler(BaseHandler):
+    def handle(self):
+        for region in list(self.regions):
+            aws_elbv2_client = boto3.client("elbv2", region_name=region)
+            logger.info(f"List Load Balancers, region: {region}")
+            self.next_token = "" if self.next_token is None else self.next_token
+            while self.next_token is not None:
+                list_stacks_params = self.selector_aws.get("list_parameters", {})
+                if self.next_token:
+                    list_stacks_params["NextToken"] = self.next_token
```

Comment:
From a quick check NextToken doesn't exist in this API , 
instead we need to use Marker and NextMarker for pagination 

https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/elbv2/client/describe_load_balancers.html